go.mod

```diff
@@ -5,11 +5,8 @@ go 1.15
 require (
 	github.com/container-storage-interface/spec v1.3.0
 	github.com/golang/mock v1.4.3
-	github.com/golang/protobuf v1.4.2 // indirect
-	github.com/kubernetes-csi/csi-lib-utils v0.7.0
+	github.com/kubernetes-csi/csi-lib-utils v0.9.0
```

Comment:
can you also check that all uses of klog in this repo are switched to klog v2?

Reply:
It seems not using klog v1 anywhere in the code.